cpp/src/Slice/PythonUtil.cpp

```diff
@@ -222,9 +222,44 @@ lookupKwd(const string& name)
     //
     static const string keywordList[] =
     {
-        "None", "and", "assert", "break", "class", "continue", "def", "del", "elif", "else", "except", "exec",
-        "finally", "for", "from", "global", "if", "import", "in", "is", "lambda", "not", "or", "pass",
-        "print", "raise", "return", "self", "try", "while", "yield"
+        "False",
+        "None",
+        "True",
+        "_",
+        "and",
+        "as",
+        "assert",
+        "async",
+        "await",
+        "break",
+        "case",
```

Comment:
I don't see `case` among the keywords for Python 2.x or Python 3.x, why did you add it here?

Reply:
case isa keyword used in match statements. It's a soft keyword; it can be used as an identifier only outside a match statement. However, adding it seemed like a good idea since I didn't want that to possibly become a problem should slice2py's Python3 support use match statements in future.